cypress/support/assertions.js

```diff
@@ -0,0 +1,24 @@
+function assertions(_chai, _utils) {
+  _chai.Assertion.addMethod('within_viewport', function withinViewport() {
```

Comment:
There are a fair few custom utility assertions that I've used previously that would be neat to have especially as we expand our test suite - this is one of them